programs/survival/eyeremover.sc

```diff
@@ -0,0 +1,17 @@
+__config() ->
+(
+	return(
+		m(
+			l('scope','player'),

```

Comment:
I suggest not specifying scope player if its default, also stay loaded - only let people add that if they need it.
In this case we don't need

Reply:
Makes sense. That just ends up in all my programs for when they get more complex. Issue been fixed.